tests/queries/0_stateless/01610_client_spawn_editor.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+#\
```

Comment:
Do I understand correctly, this file is a source code in both `bash` and `expect`?

Reply:
Yes...  And it still doesn't work in CI. It works fine on my local box.

Reply:
Let's rewrite the test without this trick... maybe it will help.